src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/ov_executable_network/properties.cpp

```diff
@@ -16,8 +16,6 @@ const std::vector<ov::AnyMap> inproperties = {
 
 const std::vector<ov::AnyMap> auto_batch_inproperties = {
     {ov::num_streams(-100)},
-    {{CONFIG_KEY(AUTO_BATCH_DEVICE_CONFIG), std::string(CommonTestUtils::DEVICE_CPU) + "(4)"},
-     {ov::auto_batch_timeout(-1)}},
```

Comment:
remove the test with `ov::auto_batch_timeout(-1)`, cause the type of this variable is unsigned int. The original code can work is because the -1 will be converted to 4294967295, and 4294967295 exceed the limitation of `std::stoi()` which throw exception. The behavior is not expected.

Reply:
this case is testing incorrect property,is not it?